src/frontend/src/binder/expr/function.rs

```diff
@@ -80,6 +84,19 @@ impl Binder {
         }
     }
 
+    /// Make sure inputs only have 2 value and rewrite the arguments.
+    /// Nullif(expr1,expr2) -> Nullif(expr1,Equal(expr1,expr2)).
```

Comment:
This is not true... It apparently should not be a recursive function...Why do you need to rewrite here? Can't we just use the original form?

Another alternative is to rewrite nullif into `CASE WHEN a==b THEN null ELSE a`. The code will be much less. Since it totally reuses the execution of case when.

Reply:
Let me see, case function now not support null, I will make case support null in new pr.